docker-compose.yml

```diff
@@ -19,6 +19,15 @@ services:
       context: .
       dockerfile: Dockerfile.test
     command: ["./scripts/test_e2e.sh"]
+  prometheus:
+    image: prom/prometheus:latest
+    ports:
+      - 9090:9090 
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
```

Comment:
Is this the best place to put the yaml file in the container? Also, where should i put prometheus.yml in the source code?